bundles/org.openhab.binding.freeboxos/src/main/resources/OH-INF/config/bridge-config.xml

```diff
@@ -42,6 +42,12 @@
 			<advanced>true</advanced>
 			<default>15682</default>
 		</parameter>
+		<parameter name="wsReconnectInterval" type="integer" min="0" max="1440" required="false" unit="min">
+			<label>Websocket Reconnect Interval</label>
+			<description>Disconnection interval frequency - 0 disables websocket usage</description>
```

Comment:
Please mention the unit in the description, it is not obvious that this is a value in minutes.

Reply:
Done